observium/Dockerfile

```diff
@@ -63,11 +63,6 @@ RUN mkdir -p /opt/observium/firstrun /opt/observium/logs /opt/observium/rrd /con
     tar zxvf observium-community-latest.tar.gz && \
     rm observium-community-latest.tar.gz
 
-# Patch schema upgrade SQL
-COPY patches/update/184.sql /opt/observium/update
```

Comment:
The patch was needed to[ get the migration to work in MariaDB](https://github.com/charlescng/docker-containers/issues/3). Has that been "fixed" in Observium?

Reply:
I pulled down the latest build and it looks like it should work with MariaDB. 👍